app/frontend/public/index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title><%= process.env.VUE_APP_TITLE %></title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Sans:100,300,400,500,700,900">
```

Comment:
i'm ok if we dont include noto-sans font as well in the html header.
it's only used in the bcgov header. let me know if you'd rather take this out

Reply:
Are noto-sans and roboto not used anywhere then? We can take them out to save the loading if so.

Reply:
- i put in noto-sans for the .gov-header .title (as they do on the bcgov design site).
- roboto is the vuetify default. I wonder if we might want it for some 'native' vuetify components
i'm not sure what the best thing to do here is. as a compromise i could just remove the noto-sans font that is only used in the header. this looked ok before in roboto.

Reply:
Ok sounds good then.